### ISSUE TITLE
hard to get a lot of netherack

### ISSUE THREAD

**Issue by ImaGooat** [jsorrell/CarpetSkyAdditions#166]
i cannot get large amounts of netherack without many hours of repetitive grinding, or without building a redstone machine that would be way to complex for me, there should be some way to get large amounts of netherack. Maybe something including cobblestone, because the official minecraft handbooks states that netherack is the nether equivalent of cobblestone

**Comment by chuckarrow** [comment]
Fair point, but the point of this mod is to make survival hard, and to push your thinking into building something amazing to do it automatically.

**Comment by Parseval1** [comment]
Using Redstone machine to obtain netherrack is not too difficult if you are not looking for efficiency. I am using such a machine. Before 1.21 you had to use players to generate new netherracks, after 1.21 it will be much easier because other entities can generate new nether portal as well
![netherrack](https://github.com/user-attachments/assets/2716bec3-e03e-4a56-b1b1-0445f5df7cd1)
![nether](https://github.com/user-attachments/assets/b14013a0-b238-4b9c-9741-7cb23b7ea3f8)

**Comment by yhs0602** [comment]
@Parseval1 What is the purpose of the anvils on the obsidians?

**Comment by Parseval1** [comment]
> @Parseval1 What is the purpose of the anvils on the obsidians?

I used the carpet to prevent the new Nether Portal from spawns on top of the original Nether Portal, but the tnt used to destroy the Nether Portal destroyed the carpet as well. So the anvils were used to raise the carpet away from the blast. This can also be replaced if there is a new way to prevent the Nether Portal from spawning on top of the original Nether Portal

**Comment by jittb** [comment]
> > 黑曜石上的铁砧有什么用？
> 
> 我用地毯来防止新的下界传送门在原来的下界传送门上生成，但是用来摧毁下界传送门的 tnt 也摧毁了地毯。因此，铁砧被用来将地毯抬高，使其远离爆炸。如果有新的方法可以防止下界传送门在原来的下界传送门上生成，也可以替换它

just need place a block over obsidian at the top
